src/Fixer/FunctionNotation/ImplodeCallFixer.php

```diff
@@ -75,7 +75,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
         $functionsAnalyzer = new FunctionsAnalyzer();
 
-        foreach ($tokens as $index => $token) {
+        for ($index = \count($tokens) - 1; $index > 0; --$index) {
```

Comment:
Any improvement here?

Reply:
Without the change the fixer fails on some PHP versions, but not on all. I didn't find the root cause for this difference. However iterating over the tokens set from start till end using `foreach` while manipulating it (in front of you) typically doesn't end well, but again, I didn't get to the bottom of it...